content/post/council-meeting-2022-07-18/chart.csv

```diff
@@ -1,26 +1,25 @@
+https://a2gov.legistar.com/LegislationDetail.aspx?ID=5723495&GUID=84DB57E4-5CF7-41B4-BF91-CBCBD69C9FE9&G=505481FE-B2ED-4B34-9A92-FDB26536D1E1&Options=&Search=,resolution amendment,DC-2 - Motion to Amend by Griswold,"Resolution to Organize a Joint Meeting of Interested Individuals from the Environmental Commission, Transportation Commission, Energy Commission, Commission on Disability Issues, A2ZERO Ambassadors, and other Community Stakeholders to Identify Opportunities for Greater Coordination around Transportation, Pedestrian Safety, and Sustainability Initiatives Related to Sidewalk Expansion and Maintenance",,,,,,,,,,,
+https://a2gov.legistar.com/LegislationDetail.aspx?ID=5723495&GUID=84DB57E4-5CF7-41B4-BF91-CBCBD69C9FE9&G=505481FE-B2ED-4B34-9A92-FDB26536D1E1&Options=&Search=,resolution amendment fail,DC-2 - Motion to Amend by Griswold,"Resolution to Organize a Joint Meeting of Interested Individuals from the Environmental Commission, Transportation Commission, Energy Commission, Commission on Disability Issues, A2ZERO Ambassadors, and other Community Stakeholders to Identify Opportunities for Greater Coordination around Transportation, Pedestrian Safety, and Sustainability Initiatives Related to Sidewalk Expansion and Maintenance",FALSE,FALSE,FALSE,FALSE,TRUE,FALSE,Recused,Absent,TRUE,FALSE,FALSE
+https://a2gov.legistar.com/LegislationDetail.aspx?ID=5723495&GUID=84DB57E4-5CF7-41B4-BF91-CBCBD69C9FE9&G=505481FE-B2ED-4B34-9A92-FDB26536D1E1&Options=&Search=,resolution amendment pass,DC-2 - Motion to Amend by Briggs,"Resolution to Organize a Joint Meeting of Interested Individuals from the Environmental Commission, Transportation Commission, Energy Commission, Commission on Disability Issues, A2ZERO Ambassadors, and other Community Stakeholders to Identify Opportunities for Greater Coordination around Transportation, Pedestrian Safety, and Sustainability Initiatives Related to Sidewalk Expansion and Maintenance",TRUE,TRUE,TRUE,TRUE,TRUE,TRUE,TRUE,Absent,TRUE,TRUE,TRUE
+https://a2gov.legistar.com/LegislationDetail.aspx?ID=5723495&GUID=84DB57E4-5CF7-41B4-BF91-CBCBD69C9FE9&G=505481FE-B2ED-4B34-9A92-FDB26536D1E1&Options=&Search=,resolution pass,DC-2 - Motion to Approve as Amended by Griswold,"Resolution to Organize a Joint Meeting of Interested Individuals from the Environmental Commission, Transportation Commission, Energy Commission, Commission on Disability Issues, A2ZERO Ambassadors, and other Community Stakeholders to Identify Opportunities for Greater Coordination around Transportation, Pedestrian Safety, and Sustainability Initiatives Related to Sidewalk Expansion and Maintenance",TRUE,TRUE,TRUE,FALSE,TRUE,TRUE,TRUE,Absent,TRUE,TRUE,TRUE
```

Comment:
ah DC-2 is also after all the amendments, we probably need to somehow get the main motion to appear before any amendment shenanigans

Reply:
That's how it works in Legistar and on the published minutes. It's done in chronological order. Reordering is probably possible, but fraught. For example, if a postponement or tabling passes, then in a sense there won't **be** a main motion anymore. I guess we could show it crossed out or something.